lisa/transformers/kernel_source_installer.py

```diff
@@ -114,22 +120,59 @@ def install(self) -> str:
         # modify code
         self._modify_code(node=node, code_path=code_path)
 
-        self._build_code(node=node, code_path=code_path)
+        config_type = source.get_config_type()
+        self._build_code(node=node, code_path=code_path, config_type=config_type)
 
         self._install_build(node=node, code_path=code_path)
 
-        result = node.execute(
-            "make kernelrelease 2>/dev/null", cwd=code_path, shell=True
-        )
+        kernel_version = ""
+        if self.__is_dom0(node) and config_type == "dom0":
```

Comment:
any other value for config_type, besides dom0?

Reply:
Hi Lili, as of now it is not. This is to identify which kernel config we should refer to. If nothing is passed then it will execute older flow. If we are building dom0 kernel, we will have to refer different file as .config

Reply:
I have made changes to user path instead of flag like dom0. Please refer the latest push.